RevenueCatUI/Data/TestData.swift

```diff
@@ -12,10 +12,8 @@ import SwiftUI
 // swiftlint:disable type_body_length file_length
 
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
-// @PublicForExternalTesting
-enum TestData {
+internal enum TestData {
```

Comment:
TestData was `public` but really didn't have a reason to be 🤔 So, removing that visibility...

Reply:
Right, I believe we have a preprocessor to make anything annotated with this public https://github.com/RevenueCat/purchases-ios/blob/dbcfe0ceb55879c98d7069c2c788af90b76b27d7/Tests/TestingApps/PaywallsTester/Preprocessor.sh#L13. 

As @vegaro mentioned, I think this is used in the paywall tester app. We did it this way because we didn't want to have this public in the SDK but to upload to App store connect, we couldn't use the `@testable` import.

Reply:
Oh interesting! I had no idea this preprocessor existed 😅 I will put this back then